nucliadb_models/src/nucliadb_models/search.py

```diff
@@ -1066,6 +1066,13 @@ def fields_validator(cls, fields) -> Self:
         return fields
 
 
+class FullResourceApplyTo(BaseModel):
+    exclude: list[str] = Field(
```

Comment:
exclude_with_labels ?

Reply:
It was defined like this in the story, let me ask to our API master :)